src/events.ts

```diff
@@ -14,7 +14,10 @@ export type DeviceConnectedEvent = CommonDeviceEvent<"device.connected">
 export type DeviceDisconnectEvent = CommonDeviceEvent<
   "device.disconnected",
   {
-    error_code: "hub_disconnected" | "device_disconnected"
+    error_code:
+      | "account_disconnected"
```

Comment:
This is probably fine I think. An alternative would be to add an `affected_device_ids` array to the `connected_account.disconnected` event. What I like about that alternative is that it correlates to the actual event that occurred here - i.e., a connected account got disconnected, and that affects N devices. It also means 1 event is generated for that case (which matches what happened here), and not N events, where N devices got disconnected.

I guess for the status of the device itself, it would be more accurate to have an "unknown" online status for this case of the account getting disconnected. We sort of did something similar with device battery levels - when the device is offline, we don't return a battery level, and in the dashboard we treat that as unknown.